traveler_core_kit/src/androidTest/java/com/guestlogix/travelercorekit/TravelerTest.java

```diff
@@ -317,7 +317,7 @@ public void onCatalogSuccess(Catalog catalog) {
                 Assert.assertNotNull(product);
                 productIdReference.reference = product.getId();
 
-                Traveler.fetchWishlist(new WishlistQuery(0, 10, new Date(), new Date()), 0, step3WishlistFetchCallback);
+                Traveler.fetchWishlist(new WishlistQuery(0, 1000, new Date(), new Date()), 0, step3WishlistFetchCallback);
```

Comment:
magic numbers makes code hard to understand especially for a new comer. even a local constant with a well defined name helps a lot

Reply:
updated